### ISSUE TITLE
Crash on clicking "vector" in style palette

### ISSUE THREAD

**Issue by tvendelin** [opentoonz/opentoonz#3516]
**Describe the bug**
When I create a vector level, select brush tool and click "vector" in style panel, Opentoonz crashes.

**To Reproduce**
List steps to reproduce the behavior.
1. Launch Opentoonz
2. Create vector level
3. Select brush tool
4. click "vector" in style palette
5. crash happens

```
~> opentoonz 
plugin search directory:/home/me/.config/OpenToonz/stuff/plugins
walkDirectory_: /home/me/.config/OpenToonz/stuff/plugins
===== PluginLoadController::finished() =====
plugin search directory:/home/me/.config/OpenToonz/stuff/plugins
walkDictionary: /home/me/.config/OpenToonz/stuff/plugins [dry]
QWidget::setMinimumSize: (XSheetToolbar/XsheetGUI::XSheetToolbar) Negative sizes (-5,30) are not possible
QWidget::setMaximumSize: (XSheetToolbar/XsheetGUI::XSheetToolbar) Negative sizes (-5,30) are not possible
/home/abuild/rpmbuild/BUILD/opentoonz-1.4.0/toonz/sources/toonz/iocommand.cpp:loadScene Progressing:
/home/abuild/rpmbuild/BUILD/opentoonz-1.4.0/toonz/sources/toonz/iocommand.cpp:loadScene end load:
/home/abuild/rpmbuild/BUILD/opentoonz-1.4.0/toonz/sources/toonz/iocommand.cpp:loadScene loadScene() completed :
create context:0x7fa394005d80 [thread:0xce71b700]
create context:0x7fa3b0010610 [thread:0xd763980]
Segmentation fault (core dumped)
```

That doesn't say a whole lot, so let me know if any additional information can /needs to be provided.

```
zypper info opentoonz 
Information for package opentoonz:
----------------------------------
Repository     : openSUSE-Leap-15.2-1
Name           : opentoonz
Version        : 1.4.0-lp152.2.1
Arch           : x86_64
Vendor         : openSUSE
Installed Size : 49.8 MiB
Installed      : Yes
Status         : up-to-date
Source package : opentoonz-1.4.0-lp152.2.1.src
```

Same happens with Morevna AppImage.

**Expected behavior**
Not crashing

**Device Information:**
 - OS: Opensuse Leap 15.2
 - OpenToonz Version: OpenToonz  v1.4.0
 - GPU Driver Version: 
```
Information for package x11-video-nvidiaG04:
--------------------------------------------
Repository     : cuda-opensuse15-x86_64
Name           : x11-video-nvidiaG04
Version        : 455.23.05-0
Arch           : x86_64
Vendor         : 
Installed Size : 196.8 MiB
Installed      : Yes (automatically)
Status         : up-to-date
Source package : x11-video-nvidiaG04-455.23.05-0.nosrc
Summary        : NVIDIA graphics driver for GeForce 400 series and newer
Description    : 
    This package provides the closed-source NVIDIA graphics driver
    for GeForce 400 series and newer GPUs.
```

**Comment by manongjohn** [comment]
I can confirm this.  It is a Linux bug only.

It crashes when trying to load .PLI files from either the Custom Styles or Vector Brush folders.

**Comment by barrysampson** [comment]
This sounds like a duplicate of [this issue](https://github.com/opentoonz/opentoonz/issues/3214).

**Comment by RodneyBaker** [comment]
Thanks @barrysampson   Will close as a duplicate.
It should be noted however that in the case I specify in that other report I was on WIndows and there is no question but that it related to me creating a custom vector brush.

**Comment by barrysampson** [comment]
@RodneyBaker I think your case in the other issue was different to the original opened by @krop and to the rest of us who posted there as we were all using Linux. I use Mac, Windows and Linux and I can reliably recreate the issue on Linux but not on the other OSes. 

At the end of the other issue I described how I traced the problem to the fixes made to solve a similar issue on MacOS by rewriting the CustomStyleManager function to use QOpenGLFramebufferObject. There are known issues with QOpenGLFramebufferObject and Nvdia drivers on Linux.
